core/pkg/resourcehandler/filesloader.go

```diff
@@ -37,19 +37,62 @@ func (fileHandler *FileResourceHandler) GetResources(sessionObj *cautils.OPASess
 	// map resources based on framework required resources: map["/group/version/kind"][]<k8s workloads ids>
 	k8sResources := setK8sResourceMap(sessionObj.Policies)
 	allResources := map[string]workloadinterface.IMetadata{}
-	workloadIDToSource := make(map[string]reporthandling.Source, 0)
 	ksResources := &cautils.KSResources{}
 
-	workloads := []workloadinterface.IMetadata{}
-
 	if len(fileHandler.inputPatterns) == 0 {
 		return nil, nil, nil, fmt.Errorf("missing input")
 	}
-	path := fileHandler.inputPatterns[0]
+
+	logger.L().Info("Accessing local objects")
+	cautils.StartSpinner()
+
+	for path := range fileHandler.inputPatterns {
+		workloadIDToSource, workloads, err := GetResourcesFromPath(fileHandler.inputPatterns[path])
+		if err != nil {
+			return nil, allResources, nil, err
+		}
+		if len(workloads) == 0 {
```

Comment:
Why are we exiting the function if the file is not a workload? What if the input contains a non-kubernetes file? We should simply ignore it.
I will print a debug message that the file is being ignored.